Cargo.lock

```diff
@@ -198,7 +198,7 @@ dependencies = [
  "iana-time-zone",
  "num-traits",
  "serde",
- "windows-targets",
+ "windows-targets 0.48.5",
```

Comment:
This seems not great. Would be good for deps if we can resolve in a way to not require multiple windows-targets.

Reply:
hmm. they're dev-deps of tracy-client-sys when it's being tested under loom. I agree I don't love the duplication in the lockfile but none of this is making it into any actual builds. I've tried updating quite a lot of things-that-cause-the-dependency and ultimately I get blocked on one that's just quite out of date, textplots -> drawille -> colored -> windows-sys 0.48 -> windows-targets 0.48.